src/js/components/TaskTable.js

```diff
@@ -100,6 +102,15 @@ class TaskTable extends React.Component {
         render: ResourceTableUtil.renderUpdated,
         sortable: true,
         sortFunction
+      },
+      {
+        className,
```

Comment:
can we cache this cell? (look on line 87 for `cacheCell` usage)

times when we can't cache are when the inputs are the same, but some external thing may change the render. for example, if you're rendering the task name in certain colors based on the status, then you can't cache.

but i believe the version cell is cacheable so we can safely cache it :)